### PR TITLE
build: migrate to dlint

### PR DESCRIPTION
**Pull request by bartlomieju**
Ref https://github.com/denoland/deno/issues/8122

Depends on https://github.com/denoland/deno_third_party/pull/78

### CODE REVIEW

**Comment by bartlomieju** [comment]
Down to 3 errors:
```
dlint (513 files)
error[no-undef]: SignalStream is not defined
   --> /Users/biwanczuk/dev/deno/cli/rt/40_signals.js:165:16
    |
165 |     return new SignalStream(signo);
    |                ^^^^^^^^^^^^
    |
error[getter-return]: Expected 'body' to always return a value.
   --> /Users/biwanczuk/dev/deno/op_crates/fetch/26_fetch.js:771:5
    |
771 |       get body() {
    |  _____^
772 | |       if (this._stream) {
773 | |         return this._stream;
774 | |       }
775 | |
776 | |       if (!this._bodySource) {
777 | |         return null;
778 | |       } else if (this._bodySource instanceof ReadableStream) {
779 | |         this._stream = this._bodySource;
780 | |       } else {
781 | |         const buf = bodyToArrayBuffer(this._bodySource);
782 | |         if (!(buf instanceof ArrayBuffer)) {
783 | |           throw new Error(
784 | |             `Expected ArrayBuffer from body`,
785 | |           );
786 | |         }
787 | |
788 | |         this._stream = new ReadableStream({
789 | |           start(controller) {
790 | |             controller.enqueue(new Uint8Array(buf));
791 | |             controller.close();
792 | |           },
793 | |         });
794 | |       }
795 | |
796 | |       return this._stream;
797 | |     }
    | |______^
    |
    = help: Return a value from the getter function
error[no-undef]: HttpClient is not defined
   --> /Users/biwanczuk/dev/deno/op_crates/fetch/26_fetch.js:878:16
    |
878 |     return new HttpClient(opCreateHttpClient(options));
    |                ^^^^^^^^^^
    |
Found 3 problems
```

Tracked in https://github.com/denoland/deno_lint/issues/462 and https://github.com/denoland/deno_lint/issues/463

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [submodule "deno_third_party"]
 	path = third_party
-	url = https://github.com/denoland/deno_third_party.git
+	url = https://github.com/bartlomieju/deno_third_party.git
```
Comment (ry) [inline .gitmodules:3]: undo

Comment (bartlomieju) [reply .gitmodules:3]: Yes, I will before landing

**Review by ry — COMMENTED** [review]
I tried it locally. Wow. It's very fast.

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**